File: MultiFiles_MergerByCPP/mergerMain.cpp
#include "../MultiFiles/pch.h"
#pragma comment(lib,"MultiFiles.lib")

int main() {
	MultiFilesMerger multiFilesMerger;
	//vector<string> toMerge = { "MultiFiles_ToMerge_SmallFiles/00000001.txt", "MultiFiles_ToMerge_SmallFiles/00000002.txt" };
	vector<string> toMerge;
	char thisFileName[256];
	for (int i = 1; i <= 1000000; i++) {
		sprintf_s(thisFileName, "MultiFiles_ToMerge_SmallFiles/%08d.txt", i);
		toMerge.push_back(thisFileName);
	}
	multiFilesMerger.merge(toMerge);
	return 0;
}

File: MultiFIles_MergerByPy/MergerMain.py

import MultiFiles

print(MultiFiles.MultiFiles)
multiFilesMerger = MultiFiles.MultiFilesMerger()
print(multiFilesMerger)
print(multiFilesMerger.merge(toGenFiles=["MultiFiles_ToMerge_SmallFiles/00000001.txt", "MultiFiles_ToMerge_SmallFiles/00000002.txt"]))